src/manager.cpp

```diff
@@ -221,123 +221,38 @@ int Manager::updateKeyword(const types::Path i_vpdPath,
         return -1;
     }
 
+    types::Path l_fruPath;
     nlohmann::json l_sysCfgJsonObj{};
 
     if (m_worker.get() != nullptr)
     {
         l_sysCfgJsonObj = m_worker->getSysCfgJsonObj();
-    }
-
-    types::Path l_fruPath = i_vpdPath;
-    types::Path l_inventoryObjPath;
-    types::Path l_redundantFruPath;
-
-    if (!l_sysCfgJsonObj.empty())
-    {
         try
         {
-            // Get hardware path from system config JSON.
-            const types::Path& l_tempPath =
-                jsonUtility::getFruPathFromJson(l_sysCfgJsonObj, i_vpdPath);
-
-            if (!l_tempPath.empty())
+            // Get the EEPROM path
+            if (!l_sysCfgJsonObj.empty())
```

Comment:
Why a try catch is needed in case "l_sysCfgJsonObj" is empty.
This should be the parent check and try catch should be nested under it.